legacyfabric-api/1.8/src/testmod/java/net/legacyfabric/fabric/test/client/rendering/ModelFeatureTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020 - 2024 Legacy Fabric
+ * Copyright (c) 2016 - 2022 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.legacyfabric.fabric.test.client.rendering;
+
+import net.fabricmc.api.ClientModInitializer;
+
+public class ModelFeatureTest implements ClientModInitializer {
+	@Override
+	public void onInitializeClient() {
+		//		LivingEntityFeatureRendererRegistrationCallback.EVENT.register((entityClass, entityRenderer, registrationHelper) -> {
```

Comment:
what's up with this?

Reply:
1.8 have generics stripped. The api is built against 1.8.9 with generics, can't use in 1.8 where they don't exist which somehow we never caught on before now.

Reply:
you can delete this then

Reply:
Done